installation/docker-containers/high-availability-install/README.md

```diff
@@ -0,0 +1,68 @@
+# Guide to install Rocket.Chat as HA with mongodb replicaset as backend
+
+## Install MongoDB Replicaset
+
+- Use <https://medium.com/@gargar454/deploy-a-mongodb-cluster-in-steps-9-using-docker-49205e231319#.8dfln7n8c> to setup the mongodb replicaset
+- Create oplogger user with rights on local oplog database:
+    `db.createUser({user: "oplogger", pwd: "password", roles: [{role: "read", db: "local"}]})`
+- Create rocket user to authenticate with rocketchat database:
+    `db.createUser({user: "rocket", pwd: "password", roles: [{role: "readWrite", db: "rocketchat"}]})`
+
+## Create rocket.chat docker container
+
+- Use example docker-compose.yaml in repo
+- Fill the INSTANCE_IP env with the local ip of the docker-host (in a 3 node setup this is unique on every host)
+- Fill the passwords from rocket and oplogger
+- Change the IP of every host in the example docker-compose.yaml -> extra_hosts inserts host entrys so each rocketchat-container could resolve the others by the name
+- Be sure that the instances could communicate on port 3000 with each other, this is important to see messages on other hosts directly
+
+```
+rocketchat:
+    image: rocketchat/rocket.chat:latest
+    environment:
+        - PORT=3000
+        - ROOT_URL=https://chat.domain.de
+        - MONGO_URL=mongodb://rocket:password@rocket-1:27017,rocket-2:27017,rocket-3:27017/rocketchat?replicaSet=rs0&readPreference=nearest&w=majority
```

Comment:
Out of curiosity where have you seen this done?  :)  Have you done: readPreference in mongo connection string before?

https://github.com/RocketChat/Rocket.Chat/issues/8064 <- with `w=majority` it might work okay and not run into this issue.  Though writes will be slower(albeit more consistent)

Reply:
Lets go ahead and let this go on.  :+1: It should work.  If we get any problems from this... we'll yank readPreference